drivers/wifi/eswifi/eswifi_offload.c

```diff
@@ -125,15 +148,14 @@ static int eswifi_off_bind(struct net_context *context,
 		return -EPFNOSUPPORT;
 	}
 
-	LOG_DBG("");
-
 	eswifi_lock(eswifi);
 
 	__select_socket(eswifi, socket->index);
 
+	socket->port = (u16_t)sys_be16_to_cpu(net_sin(addr)->sin_port);
```

Comment:
Casting seems unnecessary here as the value is already 16-bit.

Reply:
done